src/object_properties.cpp

```diff
@@ -171,4 +175,6 @@ void ObjectProperties::deSerialize(std::istream &is)
 	try {
 		damage_texture_modifier = deSerializeString(is);
 	} catch (SerializationError &e) {}
+	nametag_viewthrough_walls = readU8(is);
+	nametag_viewthrough_liquids = readU8(is);
```

Comment:
This doesn't work and still wouldn't work even if you put it inside the try.
You need one of these solutions:
* https://github.com/minetest/minetest/blob/master/src/particles.cpp#L56
* https://github.com/minetest/minetest/blob/master/src/client/content_cao.cpp#L1604

Reply:
Would the second one work if the server tried to set it to false?

Reply:
Used the `is.eof()` method. Did I do it properly?